### PR TITLE
Automate package publishing

### PR DESCRIPTION
**Pull request by lbalmaceda**
Makes use of our script to publish to npm/github when a new release PR is done.

### CODE REVIEW

**Review by adamjmcgrath — DISMISSED** [review]

**Comment by stevehobbsdev** [comment]
Let's review and modify the publish script here too so that it's not automatically published by Jenkins

**Comment by lbalmaceda** [comment]
@stevehobbsdev I believe that's what I did by removing this line https://github.com/auth0/lock/pull/2031/files#diff-1d3a9d1f60a0a89b519ce02545186f02868f7286c63746565e4a96e0f1cab281L104. Is that or did I miss something?

File: scripts/ci.sh [hunk 1]
```diff
@@ -101,7 +101,6 @@ new_line
 
 # Build & Release NPM
 yarn prepublish
```
Comment (stevehobbsdev) [inline scripts/ci.sh:103]: If we're not publishing, we don't need to do this step either

**Comment by stevehobbsdev** [comment]
@lbalmaceda Thanks, I had missed that in the diff 👍🏻

**Review by stevehobbsdev — APPROVED** [review]

**Pull request merged**